src/core/src/preprocess/preprocess_impls.cpp

```diff
@@ -324,7 +327,7 @@ void InputInfo::InputInfoImpl::dump(std::ostream& str,
 void OutputInfo::OutputInfoImpl::build(ov::ResultVector& results) {
     std::shared_ptr<opset8::Result> result;
     auto node = m_output_node;
-    auto start_out_node_names = node.get_tensor().get_names();
+    const auto start_out_node_names = node.get_tensor().get_names();
```

Comment:
```suggestion
    const auto& start_out_node_names = node.get_tensor().get_names();
```

Reply:
Here copy is intentional to restore names.